modules/platforms/dotnet/Apache.Ignite.Core/Impl/Binary/Marshaller.cs

```diff
@@ -152,6 +159,14 @@ public ITimestampConverter TimestampConverter
             get { return _cfg.TimestampConverter; }
         }
 
+        /// <summary>
+        /// Returns register same java type flag value.
+        /// </summary>
+        public bool IsRegisterSameJavaType()
```

Comment:
Nitpick: replace this method with a property

Reply:
Fixed